### DISCUSSION TITLE
How to install llvm@10?

### DISCUSSION THREAD

**Discussion by huanghantao** [Homebrew/discussions#5] [Everyday usage] [answered]
I found that the latest `LLVM` is 10, and I can see in https://formulae.brew.sh/formula/llvm. However, when I use the 

```bash
brew search LLVM

llvm ✔                            llvm@3.9                          llvm@4                            llvm@5                            llvm@6                            llvm@7
```

I did not see `LLVM@10`. The latest version of LLVM I actually saw was 8.

So, how do I install `LLVM@10`?

**Comment by huanghantao** [comment, marked as answer]
I forget execute `brew update`...

**Comment by filmackay** [comment]
Now we've moved to 11, should we get a @10 version added on brew?

**Reply by carlocab** [reply]
Best way to find out is to open a PR. You may be asked to justify that `llvm@10` satisfies https://docs.brew.sh/Versions.

**Reply by scarlehoff** [reply]
I think `numba` requires llvm 9 or 10... I'll try to open a PR if I find the time.

**Comment by Yamp** [comment]
Numba requires 9 or 10, I think,  that 10 may be preferable, would be nice to have it.

**Comment by Zialus** [comment]
Is there any particular reason for all llvm versions from 7 to 12 to be available except for llvm@10?

**Comment by reject-cookies** [comment]
Bump
